website/docs/docs/core/pip-install.md

```diff
@@ -10,24 +10,96 @@ You can install dbt Core and plugins using `pip` because they are Python modules
 <FAQ path="Core/install-pip-os-prereqs" />
 <FAQ path="Core/install-python-compatibility" />
 
-### Using virtual environments
+## Create a Python virtual environment
 
-We recommend using virtual environments (venv) to namespace pip modules.
+A Python virtual environment is an isolated workspace for Python projects. This prevents libraries and versions used in one project from interfering with others, making it especially helpful when working on multiple projects with differing requirements or avoiding conflicts with global Python installations.
 
-1. Create a new venv:
+The Python ecosystem offers several tools for creating isolated environments, such as [conda](https://anaconda.org/anaconda/conda), [poetry](https://python-poetry.org/docs/managing-environments/), and `venv`. Among these, `venv` has the fewest additional dependencies and has been included by default in recent Python versions for quite some time.
 
-```shell
-python -m venv dbt-env				# create the environment
-```
+`venv` will set up a Python virtual environment within the `env` folder.
+
+Users who want to run dbt locally, for example in [dbt Core](/docs/core/installation-overview) or the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation#install-a-virtual-environment) may want to install a Python virtual environment.
+
+## Prerequisites
+
+- Access to a terminal or command prompt.
+- Have [Python](https://www.python.org/downloads/) installed on your machine. You can check if Python is installed by running `python --version` or `python3 --version` in your terminal or command prompt.
+- Have [pip installed](https://pip.pypa.io/en/stable/installation/). You can check if pip is installed by running `pip --version` or `pip3 --version`.
+- Have the necessary permissions to create directories and install packages on your machine.
+
+## Install a Python virtual environment 
```

Comment:
- the "Install a Python virtual environment" section seems to imply you're installing the environment itself, which isn’t accurate.
   - suggestion: Rename it to "Set up a Python virtual environment" and use h3